examples/browser-webrtc/client/src/lib.rs

```diff
@@ -1,119 +1,86 @@
-use async_channel::bounded;
-use leptos::*;
+mod error;
+mod pinger;
+mod utils;
+
+use crate::error::PingerError;
+use futures::channel;
+use futures::StreamExt;
+use js_sys::Date;
 use pinger::start_pinger;
-use wasm_bindgen::JsCast;
+use wasm_bindgen::prelude::*;
 use wasm_bindgen_futures::spawn_local;
-use wasm_bindgen_futures::JsFuture;
-use web_sys::{window, Response};
 
-pub mod pinger;
+#[wasm_bindgen]
+extern "C" {
+    // Use `js_namespace` here to bind `console.log(..)` instead of just
+    // `log(..)`
+    #[wasm_bindgen(js_namespace = console)]
+    fn log(s: &str);
+}
+
+macro_rules! console_log {
+    // Note that this is using the `log` function imported above during
+    // `bare_bones`
+    ($($t:tt)*) => (log(&format_args!($($t)*).to_string()))
+}
```

Comment:
https://docs.rs/wasm-logger seems to be a light-weight way of logging to the console. Looking at its dependencies, we seem to have all of them already in our dependency tree :)

Reply:
Yes, I'll change this over right now actually.